integration-tests/dev-requirements.txt

```diff
@@ -0,0 +1,2 @@
+-r requirements.txt
+pytest-hot-reloading
```

Comment:
Is this anything really required for development or is it something nice to have? If it is not really requited, then could we only mention it as recommendation in README.md file?

Reply:
removed. It was an attempt to try something new in testing domain (aka hot reload). It would require more work to make it run.